discoverd/server/handler.go

```diff
@@ -305,6 +310,15 @@ func (h *Handler) serveGetLeader(w http.ResponseWriter, r *http.Request, params
 // servePing returns a 200 OK.
 func (h *Handler) servePing(w http.ResponseWriter, r *http.Request, params httprouter.Params) {}
 
+func (h *Handler) serveShutdown(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+	last_idx, err := h.Main.Close()
+	if err != nil {
+		hh.Error(w, err)
+		return
+	}
+	hh.JSON(w, 200, last_idx)
```

Comment:
The index should be wrapped up in a struct so that we can provide more data fields in the future.